attributes/default.rb

```diff
@@ -40,7 +40,7 @@
 default['rs-haproxy']['stats']['password'] = nil
 
 # Enable/Disable sticky sessions using cookie
-default['rs-haproxy']['session_stickiness'] = true
+default['rs-haproxy']['session_stickiness'] = false
```

Comment:
I believe this should be a string, 'false'  the recipe only evaluates a string, not a boolean.

Reply:
@cdwilhelm I updated it and made the change requested